.github/workflows/_build_test_upload.yml

```diff
@@ -236,7 +238,12 @@ jobs:
             CONDA_CHANNEL=pytorch-${{ needs.get_release_type.outputs.type }}
           fi
           set -x
-          anaconda -t "${CONDA_PYTORCHBOT_TOKEN}" upload ./*/torchdata-*.tar.bz2 -u "$CONDA_CHANNEL" --label main --no-progress --force
+          # TODO: Fix nightly conda upload with right token provided by DevInfra
+          if [[ ${{ needs.get_release_type.outputs.type }} == 'nightly' ]]; then
+            anaconda -t "${CONDA_NIGHTLY_PYTORCHBOT_TOKEN}" upload ./*/torchdata-*.tar.bz2 -u "$CONDA_CHANNEL" --label main --no-progress --force
+          else
+            anaconda -t "${CONDA_PYTORCHBOT_TOKEN}" upload ./*/torchdata-*.tar.bz2 -u "$CONDA_CHANNEL" --label main --no-progress --force
+          fi
```

Comment:
Use new nightly token for nightly release. Otherwise, using the original conda token